code/obj/machinery/nuclearbomb.dm

```diff
@@ -288,6 +292,11 @@
 		else if (P.proj_data.damage_type == D_PIERCING)
 			src.take_damage(damage)
 
+	proc/repair_nuke()
+		src.health += 5
```

Comment:
Consider `min(src.health+5, src.max_health)` avoid edge case where it could be healed to 154 if health was 149

Reply:
Got it, thank you!